src/app/client/src/app/modules/core/components/main-header/main-header.component.ts

```diff
@@ -89,6 +88,7 @@ export class MainHeaderComponent implements OnInit, OnDestroy {
   */
   enableSignup = true;
   exploreRoutingUrl: string;
+  currentUrl: any;
```

Comment:
why is this a type of any and not a type String?

Reply:
changed to type string